recipes/refseq_masher/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "refseq_masher" %}
+{% set version = "0.1.0" %}
+{% set sha256sum = "3d4460966f5a7994b66c2c87cfe5a0116d8b2bdf6a9b8b0e4ea57cd891e26708" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  skip: True # [py27]
+  number: 0
+  script: $PYTHON setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - refseq_masher = refseq_masher.cli:cli
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy >=1.12.1
+    - pandas >=0.20.1
+    - click
+    - mash
+test:
+  commands:
+    - LANG=C.UTF-8 refseq_masher --version
+    - LANG=C.UTF-8 refseq_masher --help
```

Comment:
Ah, right, sorry, forgot that the locale in the Linux image is `en_US.utf8`.
Try
```yaml
  commands:
    - refseq_masher --version
    - refseq_masher --help
    - LANG=C.UTF-8 refseq_masher --version  # [osx]
    - LANG=C.UTF-8 refseq_masher --help  # [osx]
```
instead.

Reply:
If https://github.com/bioconda/bioconda-recipes/pull/7426 is merged before this one, the test commands can be reverted to
```yaml
  commands:
    - refseq_masher --version
    - refseq_masher --help
```

Reply:
Already did that for you 😉